code/modules/mob/living/silicon/mommi/inventory.dm

```diff
@@ -62,23 +62,17 @@
 	update_items()
 	return 1
 
-//Attemps to remove an object on a mob.  Will not move it to another area or such, just removes from the mob.
-/mob/living/silicon/robot/mommi/remove_from_mob(var/obj/O)
-	src.u_equip(O,0)
-	if (src.client)
-		src.client.screen -= O
-	O.reset_plane_and_layer()
-	O.screen_loc = null
-	return 1
-
 /mob/living/silicon/robot/mommi/u_equip(W as obj)
```

Comment:
this proc just wasn't used at all, just hackish copypaste all over the file to try to replace it